whenchanged/whenchanged.py

```diff
@@ -77,6 +81,16 @@ def run_command(self, thefile):
         new_command = []
         for item in self.command:
             new_command.append(item.replace('%f', thefile))
+        now = datetime.now()
+        print_message = ''
+        if self.verbose_mode > 0:
+            print_message = self.paths[thefile] + ' changed'
+        if self.verbose_mode > 1:
+            print_message += ' at ' + now.strftime('%F %T')
+        if self.verbose_mode > 2:
+            print_message += '.' + now.strftime('%f') + ', running ' + " ".join(self.command)
```

Comment:
Could you add single-quotes around the command? i.e. running 'command arg1 arg2'